pywinauto/__init__.py

```diff
@@ -163,3 +163,8 @@ def top_from_point(self, x, y):
         """Get wrapper object for top level element at specified screen coordinates (x, y)"""
         top_element_info = self.backend.element_info_class.top_from_point(x, y)
         return self.backend.generic_wrapper_class(top_element_info)
+
+    def from_active(self):
```

Comment:
Why `from_active`? Is it more clear somehow than `get_active`?

Reply:
I thought of it more as connecting from X whether it's point, active, etc. Sort of like the entry into the application. I've updated it to `get_active`